Lib/test/test_ntpath.py

```diff
@@ -836,6 +836,8 @@ def test_abspath(self):
             tester('ntpath.abspath("")', cwd_dir)
             tester('ntpath.abspath(" ")', cwd_dir + "\\ ")
             tester('ntpath.abspath("?")', cwd_dir + "\\?")
+            self.assertEqual(nt._path_abspath("./con"), cwd_dir + "\\con")
+            self.assertEqual(nt._path_abspath("./Z:spam"), cwd_dir + "\\Z:spam")
```

Comment:
@eryksun, do you know why these tests are failing in the CI? They work on my local machine.